__tests__/app.test.js

```diff
@@ -42,3 +42,38 @@ describe('/api', () => {
       });
   });
 });
+describe('/api/articles/:article_id', () => {
+  test('get object describing an article using specific id', () => {
+    return request(app)
+      .get('/api/articles/3')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.author).toBe('icellusedkars');
+        expect(body.title).toBe('Eight pug gifs that remind me of mitch');
+        expect(body.article_id).toBe(3);
+        expect(body.body).toBe('some gifs');
+        expect(body.topic).toBe('mitch');
+        expect(body.created_at).toBe('2020-11-03T09:12:00.000Z');
+        expect(body.votes).toBe(0);
+        expect(body.article_img_url).toBe(
+          'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+        );
+      });
```

Comment:
Good work, checking the actual values of the specific article is correct here. I'd advise adding the expected status code into the test description also, for example `returns status 200 along with ...` so the description is even clearer.